src/view/com/composer/select-language/SelectPostLanguagesDialog.tsx

```diff
@@ -0,0 +1,418 @@
+            web({
+              width: gtMobile ? WEB_DIALOG_WIDTH : MOBILE_DIALOG_WIDTH,
+              maxWidth: gtMobile ? WEB_DIALOG_WIDTH : MOBILE_DIALOG_WIDTH,
+              position: a.relative,
+            }),
+            web(a.flex_1),
+            isNative && {
+              flexGrow: 1,
+              minHeight: '100%',
+            },
+          ]}>
+          <Toggle.Group
+            values={checkedLanguagesCode2}
+            onChange={setCheckedLanguagesCode2}
+            type="checkbox"
+            maxSelections={3}
+            label="languageSelection">
+            <Dialog.InnerFlatList
+              ref={listRef}
+              data={flatListData}
+              renderItem={({item, index}) => {
+                if (item.type === 'header') {
+                  const isAllLanguages = item.label === 'All languages'
+
+                  return (
+                    <Text
+                      style={[
+                        a.px_0,
+                        a.py_md,
+                        a.font_bold,
+                        a.text_xs,
+                        t.atoms.text_contrast_low,
+                        a.pt_2xl,
+                        isAllLanguages ? a.pt_2xl : a.pt_xl,
+                      ]}>
+                      <Trans>{item.label}</Trans>
+                    </Text>
+                  )
+                }
+                const lang = item.lang
+
+                return (
+                  <Toggle.Item
+                    key={lang.code2}
+                    name={lang.code2}
+                    label={languageName(lang, langPrefs.appLanguage)}
+                    style={[
+                      t.atoms.border_contrast_low,
+                      a.border_b,
+                      a.rounded_0,
+                      a.px_0,
+                      a.py_md,
+                    ]}>
+                    <Toggle.LabelText style={[a.flex_1]}>
+                      {languageName(lang, langPrefs.appLanguage)}
+                    </Toggle.LabelText>
+                    <Toggle.Checkbox />
+                  </Toggle.Item>
+                )
+              }}
+              ListHeaderComponent={listHeader}
+              stickyHeaderIndices={[0]}
+              contentContainerStyle={[a.gap_0, web({paddingBottom: 320})]}
+              style={[web(a.h_full_vh), isNative && a.px_lg]}
+            />
+          </Toggle.Group>
+          <View
+            style={[
+              a.absolute,
+              a.left_0,
+              a.right_0,
+              a.bottom_0,
+              isNative ? a.px_md : a.px_2xl,
+              a.pb_2xl,
+              a.z_10,
+              {
+                backgroundColor: t.atoms.bg.backgroundColor,
+                borderBottomLeftRadius: a.rounded_md.borderRadius,
+                borderBottomRightRadius: a.rounded_md.borderRadius,
+                borderWidth: 1,
+                borderColor: t.atoms.border_contrast_low.borderColor,
+              },
```

Comment:
Seems to be some sort of double border situation going on over here.

<img width="440" height="141" alt="Screenshot 2025-08-26 at 14 05 02" src="https://github.com/user-attachments/assets/fac70091-000d-4702-8209-131568d12a4b" />

You can replace this with:

```suggestion
              a.border_t,
              t.atoms.border_contrast_low,
              t.atoms.bg,
              {
                borderBottomLeftRadius: a.rounded_md.borderRadius,
                borderBottomRightRadius: a.rounded_md.borderRadius,
              },
```

Reply:
Looks like the border doesn't go all the way to the edges on native?